src/main/java/com/resend/services/emails/model/UpdateEmailOptions.java

```diff
@@ -0,0 +1,63 @@
+package com.resend.services.emails.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents a request to update emails.
+ */
+public class UpdateEmailOptions {
+
+    @JsonProperty("scheduled_at")
+    private final String scheduledAt;
+
+    private UpdateEmailOptions(Builder builder) {
+
+        this.scheduledAt = builder.scheduledAt;
+
+    }
+
+    /**
+     * Retrieves the schedule of the email.
+     *
+     * @return The schedule of the email.
+     */
+    public String getScheduledAt() {
+        return scheduledAt;
+    }
+
+    /**
+     * Creates a new builder instance to construct CreateEmailOptions.
+     *
+     * @return A new builder instance.
+     */
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /**
+     * Builder class for constructing UpdateEmailOptions instances.
+     */
+    public static class Builder {
+        private String scheduledAt;
+
+        /**
+         * Set the schedule of the email.
+         *
+         * @param scheduledAt The schedule of the email in ISO 8601 format (e.g., 2024-08-05T11:52:01.858Z).
+         * @return This builder instance for method chaining.
+         */
+        public Builder scheduledAt(String scheduledAt) {
```

Comment:
Should we instead use `withScheduledAt` in the Builder? If this is how it's done in the others Builders, I vote to keep consistent, but if this is the first, I think that using the `with` prefix makes more sense with the Builder way

Reply:
I don't use the 'with' prefix in the other builders. I would prefer to keep it this way without the 'with' prefix to keep the code consistent.